src/models.py

```diff
@@ -57,6 +58,37 @@
 ELMO_WEIGHTS_PATH = os.path.join(ELMO_SRC_DIR, ELMO_WEIGHTS_NAME)
 
 
+def build_model(args, vocab, pretrained_embs, tasks):
```

Comment:
Why is this showing as new? https://github.com/jsalt18-sentence-repl/jiant/blob/master/src/models.py#L129

 Is this not synced/merged properly?

Reply:
Hm not entirely sure :/  ..investigating